packages/hooks/src/usePlatformRealAccounts.ts

```diff
@@ -0,0 +1,21 @@
+import { useStore } from '@deriv/stores';
+
+const usePlatformRealAccounts = () => {
+    const { client, traders_hub } = useStore();
+    const { accounts } = client;
+    const { is_eu_user } = traders_hub;
+    const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
+
+    const platform_real_accounts = account_list.filter(account => {
+        const is_maltainvest = 'landing_company_short' in account && account.landing_company_short === 'maltainvest';
+
+        if (account.is_virtual) return false;
+        if (!is_eu_user) return !is_maltainvest;
+
+        return is_maltainvest;
+    });
+
+    return platform_real_accounts || [];
```

Comment:
```suggestion
    return platform_real_accounts;
```
@shayan-deriv I guess it's redundant.